source/supported-databases.txt

```diff
@@ -0,0 +1,61 @@
+.. _supported-databases:
+
+=======================================
+Supported Source Databases and Versions
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator supports migration from source databases to MongoDB. 
+
+Definition
+----------
+
+To create a Relational Migrator project, you can connect to a source database to migrate
+to MongoDB. This page provides the supported source databases. 
+
+.. note::
```

Comment:
[non-blocking suggestion] I feel like this could be a warning admonition instead of a note. Thoughts?

Reply:
Yes, thank you!